packages/react-components/tests/browser/RichTextSendBox.spec.tsx

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import React from 'react';
+import { expect } from '@playwright/experimental-ct-react';
+import { test as betaTest } from './FlavoredBaseTest';
+import { RichTextSendBox } from '../../src/components/RichTextEditor/RichTextSendBox';
+
+const formatButtonId = 'rich-text-input-box-format-button';
+
+betaTest.describe('RichTextSendBox tests', () => {
+  betaTest.skip(({ isBetaBuild }) => !isBetaBuild, 'The tests should be run for beta flavor only');
+  betaTest('RichTextSendBox should be shown correctly', async ({ mount }) => {
+    const component = await mount(
+      <RichTextSendBox
+        onSendMessage={async () => {
+          return;
+        }}
+      />
+    );
+    await component.evaluate(() => document.fonts.ready);
+    await expect(component).toHaveScreenshot('richTextSendBox-without-format-toolbar.png');
+    const formatButton = component.getByTestId(formatButtonId);
+
+    await formatButton.hover();
+    await expect(component).toHaveScreenshot('richTextSendBox-hover.png');
+
+    await formatButton.click();
+    //move mouse to the format button so the screenshots are consistent
+    await formatButton.hover();
+    await expect(component).toHaveScreenshot('richTextSendBox-with-format-toolbar.png');
```

Comment:
Could you rename screenshots name from `richTextSendBox...` to `rich-text-send-box...`. I'll rename the ones for input box